feature/experimental/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go

```diff
@@ -74,7 +74,12 @@ func TestBurstyTraffic(t *testing.T) {
 
 	// Configure DUT interfaces and QoS.
 	ConfigureDUTIntf(t, dut)
-	ConfigureQoS(t, dut)
+	//ConfigureQoS(t, dut)
+	if dut.Vendor() == ondatra.CISCO {
```

Comment:
recommend switch, instead of if/else

Reply:
xw-g@ and I discussed. Please leave a comment to  use switch if this expands to multiple if/else if. I am also fine if you want to change it now if you are editing line 77 above